### DISCUSSION TITLE
Vercel Deployment Slack Notification Configuration

### DISCUSSION THREAD

**Discussion by btbenedi** [vercel/community#781] [Idea]
I know the API contains data on the deployments that includes whether the deployment was initiated via a webhook, we use it in our Build Ignore Script.

Would it be possible to get some configuration or additional data on the Slack Notifications whether that deployment was fired via normal Git build or webhook?

**Comment by amyegan** [comment]
Thanks for bringing up this suggestion @btbenedi! I'm forwarding this feedback to our Product team.

`product/issues/517`

**Reply by btbenedi** [reply]
Any update on where this may fall, I noticed shortly after this request went in that the Vercel web UI got an update to show that now which is super helpful, but having it in the Slack notification would still be awesome

This screenshot shows "author" as Deploy Hook, but the slack notification is still showing the author as last code commit author
![image](https://user-images.githubusercontent.com/9112994/204877265-40fbbd15-f19c-4930-9a62-648609649c73.png)

**Reply by amyegan** [reply]
No news yet, but the team is working on improving notifications and updating the underlying systems that support them. That's part of the reason you're seeing new information become available in the UI 🙂

**Comment by ptbrowne** [comment]
Having the information in the Slack message on whether the deployment was issued was Deploy Hook (& which one) would be really convenient. This is especially important in an agency context, where a client can deploy via headless CMS. Having a deployment from the CMS or from a merge is really important but right now they appear the same "Merge branch 'main' into production deployed successfully".
